src/server/auth/server/api_server.go

```diff
@@ -203,7 +204,8 @@ func (a *apiServer) getEnterpriseTokenState() (enterpriseclient.State, error) {
 	resp, err := pachClient.Enterprise.GetState(context.Background(),
 		&enterpriseclient.GetStateRequest{})
 	if err != nil {
-		return 0, fmt.Errorf("could not get Enterprise status: %s", err.Error())
+		return 0, fmt.Errorf("could not get Enterprise status: %s",
```

Comment:
Using `%s` with `.Error()` is an anti-pattern.  You can use `%v` with the `error` type directly.

Reply:
Fixed